src/Core/Utilities/ServiceCollectionExtensions.cs

```diff
@@ -73,21 +73,21 @@ public static void AddSqlServerRepositories(this IServiceCollection services, Gl
 
             if (useEf)
             {
+                if (string.IsNullOrWhiteSpace(connectionString))
+                {
+                    throw new Exception($"Database provider type {provider.ToString()} was selected but no connection string was found.");
```

Comment:
You shouldn't need the `.ToString()` inside of the `{...}` formatter, it will call `.ToString()` for you.